src/Bridge/Doctrine/Orm/Util/QueryChecker.php

```diff
@@ -142,12 +142,35 @@ public static function hasOrderByOnToManyJoin(QueryBuilder $queryBuilder, Manage
 
                     if (isset($orderByAliases[$alias])) {
                         $relationship = QueryJoinParser::getJoinRelationship($join);
-                        list($parentAlias, $association) = explode('.', $relationship);
 
-                        $parentMetadata = QueryJoinParser::getClassMetadataFromJoinAlias($parentAlias, $queryBuilder, $managerRegistry);
+                        if (!class_exists($relationship)) {
```

Comment:
Hmm yes, or we can do:
- a try/catch thing
- test if relation is in the association mapping
- `strpos` for a `dot` in the realtionship